cwl_wes/config/log_config.py

```diff
@@ -1,59 +0,0 @@
-"""Function for configuring logging."""
-
-import logging
-import os
-from logging.config import dictConfig
-from typing import Optional
-
-from cwl_wes.config.config_parser import YAMLConfigParser
-
-
-# Get logger instance
-logger = logging.getLogger(__name__)
-
-
-def configure_logging(
-    config_var: Optional[str] = None,
-    default_path: str = os.path.abspath(
-        os.path.join(
-            os.path.dirname(
-                os.path.realpath(__file__)
-            ),
-            'log_config.yaml'
-        )
```

Comment:
Will this part of the code cause any problem? We are taking the absolute path here of the current file.

Reply:
This should be fine. But actually, you can remove `log_config.yaml` from _this_ repo, as well, because the config is taken from FOCA.